core/templates/dev/head/pages/exploration_editor/statistics_tab/issues_directive.html

```diff
@@ -11,18 +11,27 @@
       </div>
     </div>
     <div class="col-md-9 oppia-issues-border-gray">
-      <div ng-repeat="issue in issues">
-        <div ng-if="isIssueOnInitDisplay()" style="display: block;" id="issue<[getIssueIndex(issue)]>">
-          <div ng-if="issue.issueType == 'EarlyQuit'">
-            <early-quit-issue-directive index="getIssueIndex(issue)" issue="issue"/>
+      <div class="row">
+        <div ng-repeat="issue in issues">
+          <div ng-if="isIssueOnInitDisplay()" style="display: block;" id="issue<[getIssueIndex(issue)]>">
+            <div ng-if="issue.issueType == 'EarlyQuit'">
+              <early-quit-issue-directive index="getIssueIndex(issue)" issue="issue"/>
+            </div>
           </div>
-        </div>
-        <div ng-if="!isIssueOnInitDisplay()" style="display: none;" id="issue<[getIssueIndex(issue)]>">
-          <div ng-if="issue.issueType == 'EarlyQuit'">
-            <early-quit-issue-directive index="getIssueIndex(issue)" issue="issue"/>
+          <div ng-if="!isIssueOnInitDisplay()" style="display: none;" id="issue<[getIssueIndex(issue)]>">
+            <div ng-if="issue.issueType == 'EarlyQuit'">
+              <early-quit-issue-directive index="getIssueIndex(issue)" issue="issue"/>
+            </div>
           </div>
+          <!-- TODO(pranavsid98): Implement remaining issue types. -->
```

Comment:
Let's remove the comment in this PR since it needs to go in after the other ones anyway.

Reply:
Done.